pages/maintainer_alumni.html

```diff
@@ -0,0 +1,24 @@
+---
+layout: page
+title: "Maintainer Alumni"
+permalink: /maintainer_alumni/
+excerpt: Maintainers manage lesson amendments and upgrades for our global Carpentries Community.
+---
+
+{% include lazyload.html %}
+
+<section id="team" class="bg-light-gray">
+<div class="container">
+
+{% assign role = site.data.communities | where: "name", "Maintainer Alumni"  %}
+{% include community_profile_header.md %}
+
+<p>Check out our active Maintainer community <a href="{% link pages/maintainers.html %}">here</a>.</p>
```

Comment:
Give context for the link.

```suggestion
<p>Check out our <a href="{% link pages/maintainers.html %}">active Maintainer community</a>.</p>
```